.github/pull_request_template.md

```diff
@@ -36,3 +36,38 @@ _Please delete options that are not relevant._
 - [ ] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
 - [ ] My changes generate no new warnings
 - [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.
+
+## Project Metadata
+
+`` If there is nothing to fill with, then type: none ``
+
+Category:
+- [ ] Calculators
+- [ ] AI/ML
+- [ ] Scrappers
+- [ ] Social_Media
+- [ ] PDF
+- [ ] Image_Processing
+- [ ] Video_Processing
+- [ ] Games
+- [ ] Networking
+- [ ] OS_Utilities
+- [ ] Automation
+- [ ] Cryptography
+- [ ] Computer_Vision
+- [ ] Fun
+- [ ] Others
+
+Title: \<write scrip title here\>
+
+Folder: \<type the folder name that contains your script\>
+
+Requirments: \<type the name of text file containing the required to install python packages\>
+
+Script: \<Enter the name of the ``.py`` file\>
+
+Arguments: \<enter any arguments that the script needs but `-` separeted like: h-c-m\>
+
+Contributor: \<Enter your Github handle/username\>
+
+Description: \<Enter a one line description that describes your script\>
```

Comment:
Here, instruct the user to explain the arguments usage as discussed in this comment https://github.com/avinashkranjan/Amazing-Python-Scripts/issues/831#issuecomment-832744262

Reply:
User may put arguemnts descriptions in a seperate line, and the regex only extract the line with "Arguments" at the beginning. I can program it to check the rest of the lines till it reach the one which begins with "Contributor", but any ideas how to merge them?

Reply:
Umm, the argument description can be added in one long line without `\n`. The reviewer should check if the description in not multiline. If this doesn't work, we will modify the logics later